src/derivatives/05-diffusion-derivatives.md

```diff
@@ -0,0 +1,604 @@
+    preprocessed DWI data must not possess the same name as that of the raw
+    DWI data. It is RECOMMENDED to disambiguate through use of the key-value
+    "`_desc-preproc`".
+
+-   As per [common data types](02-common-data-types.md) for derivative data, a
+    JSON sidecar file is REQUIRED due to the REQUIRED `SkullStripped` field.
+
+Template:
+
+```Text
+<pipeline_name>/
+    sub-<participant_label>/
+        dwi/
+            <source_keywords>[_space-<space>]_desc-preproc_dwi.nii[.gz]
+            <source_keywords>[_space-<space>]_desc-preproc_dwi.bvals
+            <source_keywords>[_space-<space>]_desc-preproc_dwi.bvecs
+            <source_keywords>[_space-<space>]_desc-preproc_dwi.json
+```
+
+## Diffusion models
+
+Diffusion MRI can be modeled using various paradigms to extract more
+informative representations of the diffusion process and the underlying
+biological structure. A wide range of such models are available, each of
+which has its own unique requirements with respect to:
+
+-   The [input](#paramdef-input) parameters required in order to
+    define / constrain the way in which the model will be fit to the
+    empirical image data;
+
+-   The appropriate [data representations](#data-representations) utilised to
+    store information parameterised [by](#paramdef-fit) or
+    [from](#paramdef-derived) the model onto the filesystem;
+
+-   The requirements for encapsulation and complete representation of
+    derived [orientation information](#orientation-specification), which is
+    a key strength of diffusion MRI but presents unique challenges for
+    correct interpretation.
+
+### Parameter terminology
+
+Throughout this document, the term "parameter" is used to refer to
+multiple distinct sources of information. The distinction between
+these uses is defined thus:
+
+1.  <a name="paramdef-input">*Input*</a> parameter:
+
+    Value or non-numerical setting that influences the conformation
+    of the diffusion model to the empirical diffusion-weighted data.
+
+1.  <a name="paramdef-fit">Model *fit*</a> parameter:
+
+    Value that is the direct result of fitting the diffusion model to the
+    empirical diffusion-weighted data.
+
+1.  <a name="paramdef-derived">Model-*derived*</a> parameter:
+
+    Value that can be calculated directly from previously estimated
+    *model parameters*, without necessitating reference to
+    the empirical diffusion-weighted data.
+
+For example, consider a diffusion tensor model fit: the number of
+iterations in the optimisation algorithm would be an *input* parameter;
+the six unique diffusion tensor coefficients would be the *model fit*
+parameters; the Fractional Anisotropy (FA) would be a *model-derived*
+parameter (as it is calculated from the diffusion tensor coefficients
+rather than the image data).
+
+### File names
+
+```Text
+<pipeline_name>/
+    sub-<participant_label>/
+        dwi/
+            <source_keywords>[_space-<space>]_model-<label>_param-<param1>_model.nii[.gz]
```

Comment:
Should we change this suffix to `_mfp`, along the lines of the [ongoing work on dimensionality reduction networks](https://docs.google.com/document/d/1GTWsj0MFQedXjOaNk6H0or6IDVFyMAysrJ9I4Zmpz2E/edit#bookmark=id.24bbnfrlz01u)?

Reply:
I think this change was agreed on the dimensionality reduction BEP after we last discussed with @lestropie so Rob did know. Let's change it here too. Happy to get started when I back or Peer perhaps can?

Reply:
I've not followed discussions elsewhere RE file suffices; too overloaded. If MFP / MDP may catch on more broadly, can crossref to #69 / #68 / #50 and start a dedicated PR.